### ISSUE TITLE
Button "find in selection" should not be active, if....

### ISSUE THREAD

**Issue by Daijobou** [microsoft/vscode#83337]
Version: 1.39.2 (user setup)
Commit: 6ab598523be7a800d7f3eb4d92d7ab9a66069390
Datum: 2019-10-15T15:35:18.241Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
Betriebssystem: Windows_NT x64 10.0.18362

I have a text file with $this->template->var1 in it. If I now want to search for this section Example: "template->var1"), it marks the search as "find in selection" and finds only one hit. Just himself.

**Comment by roblourens** [comment]
I don't quite follow, can you give more details? Do you have the setting `editor.find.autoFindInSelection` set?

**Comment by Daijobou** [comment]
Thanks I have disabled it. But that autoFindInSelection working **very random**. One time its select it as "find in selection" in other identical cases not. See GIF
![selectin](https://user-images.githubusercontent.com/260343/67691226-99996400-f974-11e9-9c62-b8cc25fd05de.gif)

**Comment by rjdestigter** [comment]
Encountering the same issue I think. Hitting _Ctrl + F_ always automatically enables _Find in selection_  if I have text selected.

![image](https://user-images.githubusercontent.com/14314519/69270855-0decaf00-0b91-11ea-8439-63aaa9c58adb.png)

![image](https://user-images.githubusercontent.com/14314519/69270885-1e048e80-0b91-11ea-8ec7-27790fdd23b1.png)

**Comment by yzhang-gh** [comment]
I try making it clearer.

- I have `"editor.find.autoFindInSelection": true`, which means enabling "Find in selection" if there is text selected upon pressing <kbd>Ctrl</kbd>+<kbd>F</kbd>. It works well

  ![selected](https://user-images.githubusercontent.com/7588612/70108843-32fb1a80-1685-11ea-8d23-7737bc5484f3.gif)

- If there is no selection, the word under cursor will be used as the seed for search (I remember there was a setting but can't find it now)  
  **The problem is, even if there is no text selection, the "find in selection" button will still be activated**  
  That looks weird as it shows we are "finding in selection" while in fact there isn't any selection

  ![seed](https://user-images.githubusercontent.com/7588612/70109257-54103b00-1686-11ea-8e00-521d6e69182c.gif)

VSCode 1.40.2

**Comment by yeggles** [comment]
This issue was (for the most part) resolved with issue [#27083](https://github.com/microsoft/vscode/issues/27083#issuecomment-546539918).  I said "for the most part" because if I used the new "multiline" setting in my code-workspace file it will not work, but if you place it in the settings.json file it will work.  I tried to use it in the code-workspace file, because I have a multi-root project with multiple settings.json files and I wanted this setting to be set for all of the projects in one place.
